### ISSUE TITLE
Optional fonts in emscripten compile

### ISSUE THREAD

**Issue by craigsapp** [rism-ch/verovio#253]
All three music fonts are include in the emscripten compile.  It would be useful to be able to select only one so as to reduce the size of the JavaScript executable by ~25% (1 MB).

Or can this already be done manually by removing the fonts from the verovio/emscripten/data directory?

**Comment by lpugin** [comment]
The emscripten makefile copies all the ./data directory
https://github.com/rism-ch/verovio/blob/master/emscripten/build.sh#L88

You can remove them from there before the compilation. However, I would not mind having an option to be passed to the makefile to copy only selected fonts – if you want to add this ;-)

**Comment by craigsapp** [comment]
> You can remove them from there before the compilation. However, I would not mind having an option to be passed to the makefile to copy only selected fonts – if you want to add this ;-)

What is this?  Ransomware? :-)  I fixed the MIDI pitches that way, as I got tired of listening to the same wrong notes over and over.  I will probably do that for tempo as well, but probably not motivated to pay the ransom for fixing tied notes :-).

Probably some option for pattern exclusion (or deleting files matching the pattern after being compiled) would be good.  I don't like dealing with getopts, though.  Maybe I will rewrite the build script in PERL :-)
